### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ooeygui**
Preparing for Lightning release for Gen1 and Gen2. Removed unsupported features.

### CODE REVIEW

File: SampleApps.md [hunk 1]
```diff
@@ -16,19 +16,20 @@ permalink: /SampleApps.htm
...
-      <h2>Onboard LED</h2>
-      <p>Learn how to toggle the on-board LED on the Galileo on and off.</p>
-      <p><a class="btn btn-default" href="OnBoardLED.htm" role="button">View details &raquo;</a></p>
     </div>
     <div class="col-md-4">
-      <h2>OnBoard Thermal Sensor</h2>
-      <p>Learn how to use <code>AnalogRead()</code> to read values from the on-board thermal sensor.</p>
-      <p><a class="btn btn-default" href="OnBoardThermal.htm" role="button">View details &raquo;</a></p>
+      <h2>C++ REST SDK</h2>
+       <span class="label label-info">New</span>
```
Comment (anthony-ngu) [inline SampleApps.md:25]: This has label-info where everything else is label-default. Was this on purpose?

Comment (ooeygui) [inline SampleApps.md:25]: I intended to use default for updates and info for completely new items.

File: SampleApps.md [hunk 2]
```diff
@@ -101,20 +85,23 @@ permalink: /SampleApps.htm
   </div>
   <div class="row">
     <div class="col-md-4">
+      <h2>Win32 Networking</h2>
+      <span class="label label-default">New</span>
```
Comment (anthony-ngu) [inline SampleApps.md]: change to label-info since this is new?

File: SampleApps.md (hunk 1 above)
Comment (anthony-ngu) [inline SampleApps.md:25]: okay!

File: SampleApps.md (hunk 2 above)
Comment (ooeygui) [inline SampleApps.md]: good catch

**Pull request merged**